src/main/java/cc/hyperium/mods/playtime/TotalPlayTimeCommand.java

```diff
@@ -0,0 +1,52 @@
+package cc.hyperium.mods.playtime;
+
+import cc.hyperium.Hyperium;
+import cc.hyperium.commands.BaseCommand;
+import cc.hyperium.utils.ChatColor;
+import java.util.Collections;
+import java.util.List;
+
+public class TotalPlayTimeCommand implements BaseCommand {
+    @Override
+    public String getName() {
+        return "totalplaytime";
+    }
+
+    @Override
+    public String getUsage() {
+        return "/totalplaytime";
+    }
+
+    @Override
+    public void onExecute(String[] args) {
+        String messageToSend = ChatColor.translateAlternateColorCodes('&', "&9[PlayTime] &7Total play time: &f");
+
+        if (PlayTime.totalPlayTime / 86400 != 0) {
+            messageToSend = messageToSend.concat(String.valueOf((PlayTime.totalPlayTime / 86400))).concat("d ");
+        }
+
+        if (PlayTime.totalPlayTime / 3600 != 0) {
+            messageToSend = messageToSend.concat(String.valueOf((PlayTime.totalPlayTime / 3600) - ((PlayTime.totalPlayTime / 86400) * 24))).concat("h ");
+        }
+
+        if (PlayTime.totalPlayTime / 60 != 0) {
+            messageToSend = messageToSend.concat(String.valueOf((PlayTime.totalPlayTime / 60) - ((PlayTime.totalPlayTime / 3600) * 60))).concat("m ");
+        }
+
+        if (PlayTime.totalPlayTime != 0) {
+            messageToSend = messageToSend.concat(String.valueOf((PlayTime.totalPlayTime) - ((PlayTime.totalPlayTime / 60) * 60))).concat("s ");
+        }
```

Comment:
Use OOP. The logic in these two commands could be done with 1 method to avoid copy pastes

Reply:
the formatting of this version is easier to read
current - https://shardcoder-is-a.dead-me.me/i/lu4e.png
original & this one - https://shardcoder-is-a.dead-me.me/i/8d7b.png

Reply:
I never said it wasn't easy to read, I said the calculations you do is the same on both commands, so you could use a method to return the value

Reply:
added a utils file to resolve this issue